connection_pool/connection_pool_test.go

```diff
@@ -1269,12 +1528,13 @@ func runTestMain(m *testing.M) int {
 	var err error
 
 	instances, err = test_helpers.StartTarantoolInstances(servers, workDirs, test_helpers.StartOpts{
-		InitScript:   initScript,
-		User:         connOpts.User,
-		Pass:         connOpts.Pass,
-		WaitStart:    waitStart,
-		ConnectRetry: connectRetry,
-		RetryTimeout: retryTimeout,
+		InitScript:         initScript,
+		User:               connOpts.User,
+		Pass:               connOpts.Pass,
+		WaitStart:          waitStart,
+		ConnectRetry:       connectRetry,
+		RetryTimeout:       retryTimeout,
+		MemtxUseMvccEngine: true,
```

Comment:
Why not do the same as in `tarantool_test.go`?
```
MemtxUseMvccEngine: tarantool_version >= 2.10.0
```

Reply:
My bad, I forgot to change it for connection pool tests, I fixed it now:

https://github.com/tarantool/go-tarantool/blob/56c86e674da76fd0e2db0f58ff2f9edd16bb18a4/connection_pool/connection_pool_test.go#L1512